stripe/src/test/java/com/stripe/android/view/CardInputWidgetTest.java

```diff
@@ -361,6 +342,46 @@ public void onUpdateText_forAmExPrefix_callsSetImageResourceAndSetsLengthOnCvc()
         assertTrue(ViewTestUtils.hasMaxLength(mCvcEditText, 4));
     }
 
+    @Test
+    public void updateToInitialSizes_returnsExpectedValues() {
+        // Initial spacing should look like
+        // |---total == 500--------|
+        // |(card==190)--(space==260)--(date=50)|
+        CardInputWidget.PlacementParameters initialParameters =
+                mCardInputWidget.getPlacementParameters();
+        assertEquals(190, initialParameters.cardWidth);
+        assertEquals(50, initialParameters.dateWidth);
+        assertEquals(260, initialParameters.cardDateSeparation);
+    }
+
+    @Test
+    public void updateToPeekSize_withNoData_returnsExpectedValuesForCommonCardLength() {
```

Comment:
*le sigh*. I should add an AmEx test too. It'll have slightly different numbers.